src/Execution/MutationExecutor.php

```diff
@@ -2,25 +2,39 @@
 
 namespace Nuwave\Lighthouse\Execution;
 
+use Illuminate\Database\Eloquent\Relations\BelongsToMany;
```

Comment:
Please pay attention to the import order. `php-cs-fixer` can do that automatically.